### DISCUSSION TITLE
How to find out which IDs are currently subscribed to the channel

### DISCUSSION THREAD

**Discussion by sunnmas** [anycable/anycable#171] [Q&A]
The task is to send a notification to everyone who is currently subscribed to the channel. I have a User channel. I send notifications to it for users. There are periodic notifications in the system that I have to notify all users with. I can try to send a message to all users from the users table, but I would like to isolate only those who have subscribed to the channel at the moment. How to select all those users who are currently online? I.e. those who currently have an active connection and notify such instantly? and I'll figure out how to notify the others after they log in

**Comment by palkan** [comment]
There is no way to know subscribed users; pub/sub is about relying on abstract _streams_ instead of sending data directly.

For your use case, consider subscribing all users to the same channel/stream and broadcasting a message to it.
